setup.py

```diff
@@ -118,6 +120,7 @@ def is_cuda_version_ok():
     # Build
     cmake_args = [
         f"-DCMAKE_TOOLCHAIN_FILE={os.path.join(cfg.VCPKG_INSTALL_DIR, 'scripts', 'buildsystems', 'vcpkg.cmake') if args.with_pcl else ''}",
+        f"-DRGL_BUILD_ALL={'ON' if args.with_all else 'OFF'}",
         f"-DRGL_BUILD_PCL_EXTENSION={'ON' if args.with_pcl else 'OFF'}",
         f"-DRGL_BUILD_ROS2_EXTENSION={'ON' if args.with_ros2 else 'OFF'}",
         f"-DRGL_BUILD_ROS2_EXTENSION_STANDALONE={'ON' if args.with_ros2_standalone else 'OFF'}"
```

Comment:
```suggestion
        f"-DRGL_BUILD_PCL_EXTENSION={'ON' if args.with_pcl or args.with_all else 'OFF'}",
        f"-DRGL_BUILD_ROS2_EXTENSION={'ON' if args.with_ros2 or args.with_all else 'OFF'}",
        f"-DRGL_BUILD_ROS2_EXTENSION_STANDALONE={'ON' if args.with_ros2_standalone or args.with_all else 'OFF'}"
```
Suggestion to the previous comment.

Reply:
I have realized that there is no flag to build tests and tools, but I see no obstacles to adding them (`--with-tools`, `--with-tests`). What do you think?

Reply:
Hmm, how about:?
```python
if args.with_all:
    args.with_pcl = True
    args.with_ros2 = True
```

Reply:
It much more clear then banch of ORs.

Reply:
@msz-rai Same applies to autotape, logging level, etc. These are indirectly available through `--cmake "-D...`. Not sure if it makes sense to lay pipes through setup to every option in cmake. I'd settle down only for the most important ones from user's point of view.